src/renderer/components/FtSlider/FtSlider.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <label
+    class="pure-material-slider"
+    :for="id"
+  >
+    <input
+      :id="id"
+      v-model.number="currentValue"
+      :disabled="disabled"
+      type="range"
+      :min="minValue"
+      :max="maxValue"
+      :step="step"
+      @change="change"
+    >
+    <span>
+      {{ t('Display Label', {label: label, value: displayLabel}) }}
```

Comment:
I may be wrong but I think we still use $t in the vue templates for the components we migrated to composition api  (unless we're doing something wrong in the other components and need to update them)
```suggestion
      {{ $t('Display Label', {label: label, value: displayLabel}) }}
```